lib/src/tools/format_tool.dart

```diff
@@ -273,72 +271,6 @@ FormatExecution buildExecution(
       mode: ProcessStartMode.inheritStdio));
 }
 
-/// Builds and returns the list of inputs on which the formatter should be run.
-///
-/// These inputs are determined by expanding the [include] globs and filtering
-/// out any paths that match the expanded [exclude] globs.
-///
-/// Logs may be output in certain scenarios for debugging purposes.
-///
-/// By default these globs are assumed to be relative to the current working
-/// directory, but that can be overridden via [root] for testing purposes.
-Iterable<String> buildInputs(
-    {List<Glob> exclude, List<Glob> include, String root}) {
-  exclude ??= <Glob>[];
-  include ??= [
-    if (exclude.isNotEmpty) ...[
-      Glob('*.dart'),
-      Glob('benchmark/**.dart'),
-      Glob('bin/**.dart'),
-      Glob('example/**.dart'),
-      Glob('lib/**.dart'),
-      Glob('test/**.dart'),
-      Glob('tool/**.dart'),
-      Glob('web/**.dart'),
-    ],
-  ];
-  final includePaths = {
-    if (include.isEmpty) p.normalize(root ?? '.'),
-  };
-  for (final glob in include) {
-    try {
-      includePaths.addAll(glob
-          .listSync(root: root)
-          .where((entity) => entity is File || entity is Directory)
-          .map((file) => file.path));
-    } on FileSystemException catch (error, stack) {
-      _log.fine('Could not list include glob: $glob', error, stack);
-    }
-  }
-  _log.fine('Include paths:\n  ${includePaths.join('\n  ')}');
-
-  final excludePaths = <String>{};
-  for (final glob in exclude) {
-    try {
-      excludePaths.addAll(glob
-          .listSync(root: root)
-          .where((entity) => entity is File || entity is Directory)
-          .map((file) => file.path));
-    } on FileSystemException catch (error, stack) {
-      _log.fine('Could not list exclude glob: $glob', error, stack);
-    }
-  }
-  _log.fine('Exclude paths:\n  ${excludePaths.join('\n  ')}');
-
-  final excluded = includePaths.intersection(excludePaths);
-  if (excluded.isNotEmpty) {
-    _log.fine('Excluding these paths from formatting:\n  '
-        '${includePaths.intersection(excludePaths).join('\n  ')}');
-  }
-  final inputs = includePaths.difference(excludePaths);
-  if (inputs.isEmpty) {
-    _log.severe('The formatter cannot run because no inputs could be found '
-        'with the configured includes and excludes.\n'
-        'Please modify the excludes and/or includes in "tool/dev.dart".');
```

Comment:
We still want to log the messages that were previously in this function, but we'll need to do it in the `buildExecution` function instead. We should just be able to use the returned object from `getFormatterInputs`